crates/notion-core/src/serial/manifest.rs

```diff
@@ -166,3 +181,173 @@ where
         Ok(bin_map)
     }
 }
+
+#[cfg(test)]
+pub mod tests {
```

Comment:
I don't have a strong opinion either way, but you could consider splitting these out into a separate file. You'd create a `manifest/` subdirectory, move `manifest.rs` into `manifest/mod.rs`, split out this module into `manifest/test.rs`, and then change this code to

```rust
#[cfg(test)]
pub mod tests;
```

But I don't know if that's better or worse, just a thought.

Reply:
Good idea. And since you already refactored manifest.rs to manifest/mod.rs, this is an easy change